metrics/prometheus/prometheus.go

```diff
@@ -1,196 +1,197 @@
-// (c) 2021, Ava Labs, Inc. All rights reserved.
+// (c) 2025 Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
 package prometheus
 
 import (
+	"errors"
+	"fmt"
 	"sort"
 	"strings"
 
-	"github.com/ava-labs/coreth/metrics"
-
 	"github.com/prometheus/client_golang/prometheus"
 
+	"github.com/ava-labs/libevm/metrics"
+
 	dto "github.com/prometheus/client_model/go"
 )
 
-var (
-	pv             = []float64{.5, .75, .95, .99, .999, .9999}
-	pvShortPercent = []float64{50, 95, 99}
-	pvShort        = []float64{.50, .95, .99}
-)
+func init() {
+	metrics.Enabled = true
+}
```

Comment:
Needed to keep the same behavior, since we modified geth to have `Enabled = true` at global scope instead of `false`.